docs/application/dotnet/guides/app-management/widget-control.md

```diff
@@ -0,0 +1,151 @@
+
+# Widget Control
+
+The widget control provides information about installing and running widget applications.
+
+It also provides functions for the following:
+
+*   Sending update requests to the widget applications
+*   Retrieving details of running instance for the same package widget applications
+
+
+The main features of the `Tizen.Applications.WidgetControl` class include:
+
+-   [Getting information on widget applications](#getting_information)
+
+    For installed (but not necessarily running) widget applications, you can retrieve widget information with the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class.
```

Comment:
It looks like the comment to this sentence is not applied.

> Please change to:
For the widget applications that is installed but not running, you can retrieve widget information with the Tizen.Applications.WidgetControl class.

Reply:
@gotoextreme, please apply this comment.

Reply:
Done